src/main/java/com/itspace/job/endpoint/UserEndPoint.java

```diff
@@ -0,0 +1,33 @@
+package com.itspace.job.endpoint;
+
+import com.itspace.job.dto.RegisterUserDto;
+import com.itspace.job.model.User;
+import com.itspace.job.service.UserService;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserEndPoint {
+
+    private UserService userService;
+    private ModelMapper mapper;
+
+    public UserEndPoint(UserService userService, ModelMapper mapper) {
+        this.userService = userService;
+        this.mapper = mapper;
+    }
+
+    @PostMapping("/register")
+    public ResponseEntity<RegisterUserDto> register(@RequestBody User user) {
+        user = userService.register(user);
+        if (user != null) {
+            RegisterUserDto userDto = mapper.map(user, RegisterUserDto.class);
+            return ResponseEntity.ok(userDto);
+        }
+        return ResponseEntity.status(403).build();
```

Comment:
let's change this 403 http code
403 http code means forbidden
but you just need to say that something wrong for example in your case (email already exist)
let's send 400 bad request instead 403
it will be better .
In common good work, thank you
continue so !!!

Reply:
Thank you ! 
I will fix it now .